src/test/kotlin/org/rust/ide/refactoring/implementMembers/ImplementMembersHandlerTest.kt

```diff
@@ -8,9 +8,13 @@ package org.rust.ide.refactoring.implementMembers
 import com.intellij.openapi.actionSystem.ex.ActionManagerEx
 import junit.framework.TestCase
 import org.intellij.lang.annotations.Language
+import org.rust.ProjectDescriptor
 import org.rust.RsTestBase
+import org.rust.WithStdlibRustProjectDescriptor
+import org.rust.cargo.MinRustcVersion
 import org.rust.ide.inspections.RsTraitImplementationInspection
 
+@ProjectDescriptor(WithStdlibRustProjectDescriptor::class)
```

Comment:
Let's avoid setting up `WithStdlibRustProjectDescriptor` for whole test class if not all test cases require stdlib. It slows down tests without real need